src/Sylius/Behat/Context/Api/Admin/ManagingProductsContext.php

```diff
@@ -60,13 +76,34 @@ public function iRenameItToIn(string $name, string $localeCode): void
                 $localeCode => [
                     'locale' => $localeCode,
                     'name' => $name,
+                    'slug' => StringInflector::nameToSlug($name)
                 ],
             ],
         ];
 
         $this->client->updateRequestData($data);
     }
 
+    /**
+     * @When /^I set its(?:| default) price to "(?:€|£|\$)([^"]+)" for ("[^"]+" channel)$/
+     */
+    public function iSetItsPriceTo(string $price, ChannelInterface $channel): void
+    {
+        $localeCode = $channel->getLocales()->first()->getCode();
+
+        $data = [
+            'translations' => [
+                $localeCode => [
+                    'locale' => $localeCode,
+                    'price' => $price,
+                ],
+            ],
+            'channel' => '/new-api/channels/' . $channel->getCode()
```

Comment:
```suggestion
            'channel' => '/new-api/channels/' . $channel->getCode(),
```
Don't we have some rules to put commas in multiline arrays?